Pock/Widgets/Dock/DockFolder/DockFolderController/tr.lproj/DockFolderController.strings

```diff
@@ -0,0 +1,13 @@
+
+/* Class = "NSButtonCell"; title = "Open"; ObjectID = "9UV-hx-aj8"; */
+"9UV-hx-aj8.title" = "Aç";
+
+/* Class = "NSTextFieldCell"; title = "Label"; ObjectID = "JrJ-Ma-3oA"; */
+"JrJ-Ma-3oA.title" = "Label";
```

Comment:
"Label" yerine farklı bir kelime yazılamaz mı? :)

Reply:
Kodda klasör ve klasör detayı olarak tanımlanmış o label'lar. Klasöre göre değiştiği için ne yazsam görünmeyecek zaten o şekilde bıraktım.